heron/tools/cli/src/python/submit.py

```diff
@@ -366,6 +368,16 @@ def submit_cpp(cl_args, unknown_args, tmp_dir):
 
   return launch_topologies(cl_args, topology_file, tmp_dir)
 
+def download(uri):
+  current_path = os.path.realpath(__file__)
+  print current_path
+  tmp_dir = tempfile.mkdtemp()
+  subprocess.call([current_path + "downloader.sh", uri, tmp_dir])
+  suffix = (".jar", ".tar", ".tar.gz", ".pex", ".dylib", ".so")
+  for f in os.listdir(tmp_dir):
+    if f.endswith(suffix):
```

Comment:
If there are multiple files in this tmp directory, then this `download` method will return the first file it encountered ending with a valid suffix. But is this returned file truly a valid topology file?

If there will be only one file downloaded to this directory, I suggest we remove this `for` loop.

Reply:
we need to make an agreement on this with users. the users may assume there is only one file, or assume there are multiple files